cmd/release/cmd/update.go

```diff
@@ -130,11 +132,44 @@ var updateChartsCmd = &cobra.Command{
 	},
 }
 
+var updateRancherCmd = &cobra.Command{
+	Use:   "rancher",
+	Short: "Update rancher files",
+}
+
+var updateRancherDashboardCmd = &cobra.Command{
+	Use:   "dashboard [version]",
+	Short: "Update Rancher's Dashboard and UI references and create a PR",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("expected at least one argument: [version]")
+		}
+
+		version := args[0]
+		versionTrimmed, _, _ := strings.Cut(version, "-rc")
```

Comment:
Does this return an error? If so, let's check it.

Reply:
Here it return `(string, string, bool)`, the `bool` being if it found the `-rc` section to cut. If it was found or not doesn't matter for this workflow.